src/app/Topology/Shared/Entity/EntityKeyValues.tsx

```diff
@@ -17,22 +17,22 @@ import { Label, LabelGroup } from '@patternfly/react-core';
 import * as React from 'react';
 import { EmptyText } from '../EmptyText';
 
-export const EntityLabels: React.FC<{ labels?: object; maxDisplay?: number }> = ({ labels, maxDisplay, ...props }) => {
-  const _transformedLabels = React.useMemo(() => {
-    return labels ? Object.keys(labels).map((k) => `${k}=${labels[k]}`) : [];
-  }, [labels]);
+export const EntityKeyValues: React.FC<{ kv?: object; maxDisplay?: number }> = ({ kv, maxDisplay, ...props }) => {
```

Comment:
How about we add a prop called `transformFn` that is a function to transform whatever `kv` is. This way, we can reuse this component and remove `TargetPaths`?

Reply:
For the case of `TargetPaths`, the function can be undefined :))

Reply:
Probably, also need to change the type of `kv`. It could be `object | string`.

Reply:
oh yehh right thankss!!